test-arquillian-container/pom.xml

```diff
@@ -197,6 +201,45 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>org.jboss.as.plugins</groupId>
+            <artifactId>jboss-as-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>start-server</id>
+                <phase>generate-test-resources</phase>
+                <goals>
+                  <goal>start</goal>
+                </goals>
+                <configuration>
+                  <!-- It looks like JBoss Maven plugin needs absolute paths -->
+                  <jbossHome>${project.basedir}/${test.server.build.dir}</jbossHome>
```

Comment:
@rnc @pgier Unfortunately I had to make this path absolute. This is weird - wildfly plugin seems to work with both absolute and relative paths.